users-service/features/support/env.ls

```diff
@@ -8,7 +8,7 @@ require! {
 db = null
 get-db = (done) ->
   return done db if db
-  MongoClient.connect "mongodb://localhost:27017/space-tweet-users-test", N (mongo-db) ->
+  MongoClient.connect "mongodb://localhost:27017/space-tweet-db-test", N (mongo-db) ->
```

Comment:
The same _database server_ (in this case `localhost`), but different _databases_. A database is just a logical "file" inside the database server.